ci/settings.ini

```diff
@@ -2,7 +2,7 @@
 
 [aqt]
 concurrency: 2
-baseurl: https://download.qt.io
+baseurl: https://qt.mirror.constant.com
```

Comment:
I don't like to change it into mirror site.
It may be works well and only for CI process, but it is not encouraged for users.

It is better to add comment in the file why it uses mirror and the concern.

Reply:
I'm fixing this in the commit that also fixes the list-qt issue!